imaging.rst

```diff
@@ -65,14 +66,83 @@ QR Code® is a registered trademark of DENSO WAVE INCORPORATED.
 
 ----
 
+.. raw:: latex
+
+    \clearpage
+
+.. _1d286b:
+.. index:: $1D $28 $6B - Dynamic 2D Barcode
+Dynamic 2D Barcode - ``$1D $28 $6B`` |phx|
```

Comment:
I like the tag look for the product and that this now allows the permalink to properly work.